### ISSUE TITLE
where would it make sense to test running an analysis?

### ISSUE THREAD

**Issue by jdkent** [neuroscout/neuroscout-cli#113]
Since I ran into an unexpected error after updating, would it make sense to place a test in this repo for running a little analysis in continuous integration?
Or would there be a better place? Or is a test not needed?

**Comment by adelavega** [comment]
yeah that wouldn't be a bad idea. fitlins should also have more robust tests than it currently does. 

possible we could set up Circle CI, and using pyns create an analysis in neuroscout which is then run (so the whole pipeline is "fresh"). I think as long as this isn't getting triggered all the time, it would be a good test to run periodically.

**Comment by adelavega** [comment]
if you have any expertise with that, and want to help lmk

**Comment by adelavega** [comment]
also, just pinning a "stable" version will also help, but definitely testing makes sense, just haven't gotten around to it.
